docs/content/guide/controller.ngdoc

```diff
@@ -253,9 +253,9 @@ scopes being created for our view:
 
 - The root scope
 - The `MainController` scope, which contains `timeOfDay` and `name` properties
-- The `ChildController` scope, which inherits the `timeOfDay` property but overrides (hides) the `name`
-property from the previous
-- The `GrandChildController` scope, which overrides (hides) both the `timeOfDay` property defined in `MainController`
```

Comment:
TBH, I like the "hides". It gives a better "feel" of what is going on especially for people not familiar with prototypal inheritance in JS. Leaving "overrides" might give the impression that it changes the value on the parent scope as well.

"Shadows" would be a good alternative for "hides" too, imo.